src/rmem.c

```diff
@@ -0,0 +1,470 @@
+#include "rmem.h"
+
+
+typedef uintptr_t ptrcmp;
+
+static size_t _AlignSize(const size_t size, const size_t align)
+{
+    return (size + (align-1)) & -align;
+}
+
+
+/************* Memory Pool *************/
+
+static void _RemoveNode(struct MemNode **const node)
+{
+    ((*node)->prev != NULL)? ((*node)->prev->next = (*node)->next) : (*node = (*node)->next);
+    ((*node)->next != NULL)? ((*node)->next->prev = (*node)->prev) : (*node = (*node)->prev);
+}
+
+struct MemPool MemPool_Create(const size_t size)
+{
+    struct MemPool mempool = {0};
+    if( size==0UL )
```

Comment:
Please keep the coding style coherent. This should read `if (size == 0)`, same for the rest of the commits.